packages/imperative/src/imperative/src/plugins/cmd/install/install.handler.ts

```diff
@@ -148,7 +151,10 @@ export default class InstallHandler implements ICommandHandler {
                         params.response.console.log("\n_______________________________________________________________");
                         const registryInfo = NpmRegistryUtils.buildRegistryInfo(packageString, params.arguments.registry);
                         params.response.console.log("Location = " + registryInfo.location + "\n");
-                        const pluginName = await install(packageString, registryInfo);
+                        const pluginName = await install(packageString, registryInfo, false, params.arguments.verbose);
+                        if (params.arguments.verbose) {
+                            params.response.console.log("_______________________________________________________________");
+                        }
```

Comment:
I'm guessing this is to make the output a bit cleaner 😋 
Feels a bit strange though 😅

Reply:
(UX nitpick) If I make my terminal small, the separator spans across multiple rows. We could use `process.stdout.columns` to determine how long to make the separator, e.g.:

```ts
if (params.arguments.verbose) {
    params.response.console.log("_".repeat((process.stdout.columns ?? 80) * 0.5));
}
```